include/zephyr/net/wifi_mgmt.h

```diff
@@ -768,6 +777,105 @@ struct wifi_ap_config_params {
 	uint32_t max_num_sta;
 };
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_DPP
+#define WIFI_DPP_QRCODE_MAX_LEN 255
+
+enum wifi_dpp_op {
```

Comment:
You need to document these using doxygen comments `/** .... */`, same comment to other added symbols.